_posts/2022-11-30-trino-summit-2022-bloomberg-recap.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Trino at Bloomberg"
+author: "Vishal Jadhav, Pablo Arteaga, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/bloomberg.jpg
+---
+
+This post continues [a larger series of posts]({% post_url
+2022-11-21-trino-summit-2022-recap %}) on the Trino Summit 2022 sessions.
+Following the [Trino at Apple talk]({\% post_url
+2022-11-22-trino-summit-2022-apple-recap %}), engineers from Bloomberg shared
+the latest in their additions to Trino. With so many analysts producing huge
```

Comment:
Change
"With so many analysts producing huge amounts of data, Bloomberg uses Trino to federate it all together."

To
"Bloomberg uses Trino to federate huge amounts of disparate financial data together."